apps/transport/lib/transport_web/controllers/resource_controller.ex

```diff
@@ -149,10 +151,8 @@ defmodule TransportWeb.ResourceController do
     conn |> assign(:resource, resource) |> render("details.html")
   end
 
-  defp render_gtfs_details(conn, params, resource) do
-    validation = latest_validation(resource)
```

Comment:
Note to self: c'est ici qu'on supprime un recalcul (duplicate) déjà fait dans l'appelant de toute façon.